frontend/src/components/partners/currentProjects.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
```

Comment:
Can you reorder the imports here.. seperate absolute and relative imports.

Reply:
@royallsilwallz code styling improvements like this one will be on our side.